src/main/java/org/jabref/gui/openoffice/StyleSelectDialogViewModel.java

```diff
@@ -129,4 +124,8 @@ public void storePrefs() {
         preferences.setCurrentStyle(selectedItem.getValue().getStylePath());
         preferencesService.setOpenOfficePreferences(preferences);
     }
+
+    private StyleSelectItemViewModel getStyleOrDefault(String stylePath) {
+        return styles.stream().filter(style -> style.getStylePath().equals(stylePath)).findFirst().orElse(styles.get(0));
```

Comment:
What happens if styles is empty?

Reply:
Styles can never be empty because we have two internal default styles that are always available.